cmd/res/provision_watchers/llrp.provision.watcher.json

```diff
@@ -1,16 +1,14 @@
 {
   "name": "llrp-provision-watcher",
-  "identifiers": {},
+  "identifiers": {
+    "vendorPEN": "*"
```

Comment:
```suggestion
    "vendorPEN": "."
```
Just found out the value is a regular expression so `.` is the wild card.

Reply:
Made suggested change

Reply:
Haven't had the chance to verify it yet, but if this is a regex like @lenny-intel says, then shouldn't it be `.*`?

Reply:
> then shouldn't it be `.*`?

@ajcasagrande , I just went of what @cloudxxx8 demonstrated in another Device Service that is using Provision Watcher.

Reply:
Ok, then it is likely using a regex search (as opposed to full string match). In that case, this is fine.